sdks/java/core/src/test/java/org/apache/beam/sdk/io/BigQueryIOTest.java

```diff
@@ -26,8 +26,8 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertThat;
-import static org.mockito.Mockito.anyString;
-import static org.mockito.Mockito.eq;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
```

Comment:
I guess it happened on organize imports? Not particularly sure why that happened either...